package.json

```diff
@@ -21,49 +17,10 @@
   "author": "Brent Jackson",
   "license": "MIT",
   "devDependencies": {
+    "@compositor/styleguide": "0.0.8",
```

Comment:
I'm going to remove this dependency until it's public so that anyone can contribute